testing/buildcop-function/main.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import base64
+import json
+import os
+import subprocess
+from datetime import datetime, timedelta
+
+
+def write_xml_file(build_event):
```

Comment:
In reviewing this, I gave some thought to the lack of test coverage and a CI pipeline on this function. I think it's okay that we skip for now, but if we increase the complexity of this function later we should think about adding tests then.